ts/features/itwallet/identification/hooks/useItwIdpIdentification.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  LoginUtilsError,
+  openAuthenticationSession
+} from "@pagopa/io-react-native-login-utils";
+import * as O from "fp-ts/lib/Option";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import React from "react";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { isMixpanelEnabled } from "../../../../store/reducers/persistedPreferences";
+import { LocalIdpsFallback } from "../../../../utils/idps";
+import { getIdpLoginUri } from "../../../../utils/login";
+import { isFastLoginEnabledSelector } from "../../../fastLogin/store/selectors";
+import { lollipopKeyTagSelector } from "../../../lollipop/store/reducers/lollipop";
+import { regenerateKeyGetRedirectsAndVerifySaml } from "../../../lollipop/utils/login";
+import {
+  ItWalletError,
+  ItWalletErrorTypes
+} from "../../common/utils/itwErrorsUtils";
+
+const URL_LOGIN_SCHEME = "iologin";
+
+export const useItwIdpIdentification = () => {
```

Comment:
There are minor differences between this authentication method (SAML auth 2.0) and the conventional IO auth procedure. For the time being, **I advise creating a simple scaffolded hook and eliminating all logic associated with the conventional IO auth flow**. According to the specifications, the SAML auth flow begins with a PAR request and proceeds through a webview. The user eventually sees a login screen. There is no fastlogin or lollipoKeyTag involved in this flow.

Reply:
> There is no fastlogin or lollipoKeyTag involved in this flow.

So you are suggesting to remove the `regenerateKeyGetRedirectsAndVerifySaml` function from the pipe?

Reply:
> Maybe also local idps and login uri could change. In our request we pass a value named `idphint` to identify the Id Provider. In the scenario of the user auth flow we load a webview with an url GET /authorize?idphint={value}....

URL and other parameters could be easily changed later on. This PR is for the identification flow handling implementation.